source/globalCommands.py

```diff
@@ -462,27 +462,27 @@ def script_toggleReportLineIndentation(self,gesture):
 		lineIndentationSpeech = config.conf["documentFormatting"]["reportLineIndentation"]
 		lineIndentationTones = config.conf["documentFormatting"]["reportLineIndentationWithTones"]
 		if not lineIndentationSpeech and not lineIndentationTones:
-			# Translators: The message announced when toggling the report line indentation document formatting setting.
-			ui.message(_("report line indentation with speech"))
+			# Translators: The message announced when cycling through line indentation settings.

```

Comment:
Sorry to be picky, but I also meant we should change "The message" to "A message", since it's one of several messages that might be reported when cycling. While you're at it, "announced" should be "reported". Again, this applies to all of these comments.